client/app/telecom/telephony/billingAccount/billing/depositMovement/telecom-telephony-billing-account-billing-deposit-movement.html

```diff
@@ -83,7 +83,7 @@ <h4 data-translate="telephony_group_billing_deposit_movement_success"></h4>
                             </label>
                             <div class="input-group">
                                 <input type="number"
-                                       class="form-control input-amount inline-block"
+                                       class="form-control input-amount d-inline-block"
```

Comment:
You'll normally have a conflict since I replaced here `<div class="input-group">` with an `<input-number-spinner>`.

Reply:
At this time I don't